tests/unit/stream_alert_cli/test_config.py

```diff
@@ -14,6 +14,7 @@
 limitations under the License.
 """
 # pylint: disable=protected-access
+# pylint: disable=no-self-use
```

Comment:
sorry one last nit - these can be on one line separated by commas :)

`pylint: disable=protected-access,no-self-use`